.travis.yml

```diff
@@ -32,6 +32,9 @@ matrix:
     - php: 7.0
       env:
         - SERVER_VERSION=3.4.11
+    - php: 7.0
+      env:
+        - SERVER_VERSION=4.0.0-rc4
```

Comment:
Relevant: https://github.com/mongodb/mongo-php-driver/pull/851

I'm not sure this will be helpful for the transaction tests (pending [PHPC-1184](https://jira.mongodb.org/browse/PHPC-1184)), since they require a replica set oplog to exist.

Reply:
Yeah, I am aware of that. But some tests just need a 4.0.0 one to run at, as they don't actually *talk* to the Database yet. But libmongoc still does the version check with a single server.